woocommerce/woocommerce
# Remove welcome to woocommerce for store users note

### All Submissions:






### Changes proposed in this Pull Request:



Part of #34930 






### How to test the changes in this Pull Request:



1. Visit the homepage with `from_calypso` appended to the URL. `wp-admin/admin.php?page=wc-admin&from_calypso=1`
1. Install a plugin like [WP Crontrol](https://wordpress.org/plugins/wp-crontrol/)
1. Under Tools -> Cron Events, run wc_admin_daily
1. Make sure the note with a title of "Welcome to your new store management experience" has not been added



### Other information:




### FOR PR REVIEWER ONLY:
